src/network/network.cpp

```diff
@@ -1040,7 +1040,7 @@ void NetworkGameLoop()
 		/* Log the sync state to check for in-syncedness of replays. */
 		if (TimerGameCalendar::date_fract == 0) {
 			/* We don't want to log multiple times if paused. */
-			static Date last_log;
+			static TimerGameCalendar::Date last_log;
```

Comment:
lol, is this timer based on TimerGameCalendar? Oy .. that will be a problem :D But this is why we mark these things .. not for this PR, but in later PRs if we make GameCalendar scalable, this needs to be addressed. I guess it should actually be `TimerGameTick`, but again, for another PR :)